common/test/acceptance/tests/helpers.py

```diff
@@ -278,6 +279,8 @@ def setUp(self):
         super(EventsTestMixin, self).setUp()
         self.event_collection = MongoClient()["test"]["events"]
         self.event_collection.drop()
+        self.user_event_collection = MongoClient()["test"]["user_events"]
```

Comment:
Can we call this "browser_event_collection"? Both types of events are user events.